nat-lab/tests/conftest.py

```diff
@@ -244,22 +244,55 @@ def pytest_runtest_setup():
     asyncio.run(perform_pretest_cleanups())
 
 
+# pylint: disable=unused-argument
+def pytest_runtest_call(item):
```

Comment:
Isn't this called before each test? But I might be interpreting this wrong https://docs.pytest.org/en/7.1.x/reference/reference.html?highlight=pytest_runtest_call#pytest.hookspec.pytest_runtest_call

Reply:
ah nvm, I now see that pcaps are for each test